build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt

```diff
@@ -38,12 +38,11 @@ class ArtifactSizeMetricsPlugin : Plugin<Project> {
     }
 }
 
-private fun Project.subprojectArtifactSizeMetricsTask(): TaskProvider<CollectArtifactSizeMetrics> =
-    tasks.register<CollectArtifactSizeMetrics>("artifactSizeMetrics") {
-        group = TASK_GROUP
-        onlyIf { tasks.findByName("jvmJar") != null }
-        dependsOn(tasks.withType<Jar>())
-    }
+private fun Project.subprojectArtifactSizeMetricsTask(): TaskProvider<CollectArtifactSizeMetrics> = tasks.register<CollectArtifactSizeMetrics>("artifactSizeMetrics") {
+    group = TASK_GROUP
+    onlyIf { tasks.findByName("jvmJar") != null }
+    dependsOn(tasks.withType<Jar>())
+}
```

Comment:
**Question**: I'm noticing a lot of reformatting which lengthens lines past 120 chars. Was this an automated reformat or did you manually make these changes for some reason?

Reply:
This was an automated format from ktlint